app/content/views/event.py

```diff
@@ -193,3 +198,34 @@ def get_events_where_is_admin(self, request, *args, **kwargs):
         return self.paginate_response(
             data=events, serializer=EventListSerializer, context={"request": request}
         )
+
+    @action(
+        detail=True,
+        methods=["post"],
+        url_path="mail-gift-cards",
+        parser_classes=(MultiPartParser, FormParser,),
+    )
+    def mail_gift_cards(self, request, *args, **kwargs):
+
+        event = self.get_object()
+        dispatcher = request.user
+        files = request.FILES.getlist("file")
+
+        try:
+            success = send_gift_cards_by_email(event, files, dispatcher)
+            return Response(
+                {
+                    "detail": "Gavekortene er sendt! Se separat epost for en mer utfyllende oversikt.",
+                    "success": success,
+                }
+            )
+        except EventGiftCardAmountMismatchError as e:
+            return Response({"detail": e.message}, status=status.HTTP_400_BAD_REQUEST)
+        except ValueError as e:
+            capture_exception(e)
+            return Response(
+                {
+                    "detail": "Noe gikk galt da vi prøvde å sende ut gavekortene. Gi det et nytt forsøk senere eller kontakt Index på index@tihlde.com eller slack."
```

Comment:
Bruk epost konstant i constant.py
```suggestion
                    "detail": "Noe gikk galt da vi prøvde å sende ut gavekortene. Gi det et nytt forsøk senere eller kontakt Index på index@tihlde.org eller slack."
```